### PR TITLE
Bug/fix syntax highlighting

### PR DESCRIPTION
**Pull request by domharrington**
There was a bug in the syntax highlighter which meant it couldn't syntax highlight java and HTML. I've fixed that and also made sure the Examples are correctly syntax highlighting.

### CODE REVIEW

File: example/src/Demo.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ class Demo extends React.Component {
               }}
               flags={{ correctnewlines: false }}
               suggestedEdits
-              oauthUrl={this.props.oauthUrl}
+              oauth={this.props.oauth}
```
Comment (uppal101) [inline example/src/Demo.jsx]: This isn't a change from the new API key, is it?

File: packages/syntax-highlighter/codemirror.js [hunk 2]
```diff
@@ -6,14 +6,73 @@ require('codemirror/mode/shell/shell');
...
+  //   if (info.mime == lang) {
+  //     modeName = info.mode;
+  //   } else if (info.name.toLowerCase() === lang) {
+  //     modeName = info.mode;
+  //     lang = info.mime;
+  //   }
+  // });
   // if (!CodeMirror.modes[lang]) {
   //   require(`codemirror/mode/${lang}/${lang}.js`);
   // }
```
Comment (uppal101) [inline packages/syntax-highlighter/codemirror.js]: Do we need this if block it seems to be commented out from last time?

File: example/src/Demo.jsx (hunk 1 above)
Comment (domharrington) [reply example/src/Demo.jsx]: I recently renamed `oauthUrl` to `oauth` in the ApiExplorer props, but this was still using the old way.

File: packages/syntax-highlighter/codemirror.js (hunk 2 above)
Comment (domharrington) [reply packages/syntax-highlighter/codemirror.js]: There's a TODO at the top of the block linking to this: https://github.com/readmeio/api-explorer/issues/101 which we will eventually need to do.

**Pull request merged**